docs/dev/table/hive/scala_shell_hive.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Use Hive in Scala Shell"
+nav-parent_id: hive_tableapi
+nav-pos: 3
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+In order to use hive connector in [Flink Scala Shell]({{ site.baseurl }}/ops/scala_shell.html), you need to put the following jars under lib folder of flink dist.
+
+* flink-connector-hive_{scala_version}-{flink.version}.jar
```

Comment:
probably just add reference to https://ci.apache.org/projects/flink/flink-docs-master/dev/table/hive/#depedencies  and explain the scala_version and flink version?

Reply:
Fixed, I didn't explain scala_version and flink version because I think they are clear enough, user should understand what they refer to.